app/models/edition.rb

```diff
@@ -483,6 +483,10 @@ def paths
     paths
   end
 
+  def html_ref_for_error(attribute)
+    "#edition_#{attribute}"
+  end
```

Comment:
It doesn't feel quite right for this to belong to the Model / data layer of our application. It seems like more of a View / presentation layer concern.

Did you consider implementing this in a View Helper instead? There may be a good reason why that wouldn't be possible which I'm just missing context on.

Reply:
Done :-)